static/js/graphs_page.js

```diff
@@ -2609,4 +2614,6 @@ var cytoscapeGraph = {
         });
     }
 
-};
\ No newline at end of file
+};
+
+var original_opacity = {};
```

Comment:
It should be a property of layoutEditor module and should be initialized when layoutEditor is initialized.

Reply:
Moved original_opacity variable into layoutEditor model and initialized as graphPage.layoutEditor.original_opacity